x-pack/plugins/ingest_manager/server/saved_objects/migrations/to_v7_10_0.ts

```diff
@@ -92,3 +93,12 @@ export const migrateSettingsToV7100: SavedObjectMigrationFn<
 
   return settingsDoc;
 };
+
+export const migrateInstallationToV7100: SavedObjectMigrationFn<
```

Comment:
If someone upgrades form 7.9 to 7.11, I assume this will still be run?

Reply:
I certainly hope so. Kibana platform decides which migrations are run at which point in time, but I couldn't find any documentation about it. @jen-huang do you know more?

Reply:
yes it will, if there are migrations registered for a saved object type, it will run the migrations to bring the SO docs up-to-date, and mark them with a `migrationVersion` property. the value of this property is not tied to the Kibana version, but we (and many other plugins) use the Kibana version just for simplicity. so migrations doesn't care about the Kibana version, they will simply run until the SO docs are up-to-date with the latest registered migration version.

https://github.com/elastic/kibana/blob/master/src/core/server/saved_objects/migrations/README.md#example